README.md

```diff
@@ -161,7 +182,7 @@ gorc eth-to-cosmos \
     --erc20-address="0x0000000000000000000000000000000000000000" \
     --amount="1.3530000" \
     --cosmos-destination="$(sommelier keys show orchestrator -a)"
-    
+
```

Comment:
We may want to remove these from the default README, there's really no need for an individual validator to bridge assets during their setup, and the extra traffic spends ETH.